### PR TITLE
Fixed - add token

### PR DESCRIPTION
**Pull request by iosdevashok**
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-TODO)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/TODO)

### Description
TODO

### Pull requests checks where applicable
- [ ] Fill in the three TODOs above (tickets number and description of your work)
- [ ] The PR name follows our [naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Accessibility implemented and tested (minimum Dynamic Text and VoiceOver)
- [ ] Unit tests written and passing
- [ ] Documentation / comments for complex code and public methods

### CODE REVIEW

**Comment by iosdevashok** [comment]
https://github.com/pellartech/carbon-ios/assets/102407838/40c55f3e-74a3-4e65-b894-39a07a10a6a8

**Comment by madtocc** [comment]
There's no need to ask the user to switch the network. Please remove the alert and proceed changing the network.

File: Client/Frontend/Wallet/AddToken/AddCustomTokenViewController.swift [hunk 1]
```diff
@@ -804,16 +802,36 @@ class AddCustomTokenViewController: UIViewController {
     @objc func addTokenBtnTapped (){
         if ( self.tokenNetworkValueLabel.text  == selectedNetwork.name?.capitalized){
             SVProgressHUD.show()
-                let contractAddress = self.tokenInfo?.contract_address ?? ""
-                self.addToken(tokenArray: [contractAddress])
+            let contractAddress = self.tokenInfo?.contract_address ?? ""
+            self.addToken(tokenArray: [contractAddress])
         }else{
-            self.view.makeToast("Sorry! Unable to add. Please select different token or network", duration: 3.0, position: .bottom)
+            let alertVC = UIAlertController(title: "Alert!", message: "Currently, you are in \(selectedNetwork.name?.capitalized ?? ""). Do you want to switch network?", preferredStyle: .alert)
```
Comment (madtocc) [inline Client/Frontend/Wallet/AddToken/AddCustomTokenViewController.swift:808]: no need.

**Comment by iosdevashok** [comment]
https://github.com/pellartech/carbon-ios/assets/102407838/48a9bc0a-38df-40bd-b423-29e553c5e127

**Review by madtocc — APPROVED** [review]

**Pull request merged**